src/modules/layout/components/MaterialHeaderButtons.js

```diff
@@ -21,13 +21,13 @@ const onOverflowMenuPress = (cancelButtonLabel: string) => ({ overflowButtonRef,
     cancelButtonLabel
   })
 
-const MaterialHeaderButtons = (props: { cancelLabel: string, children: React.Node, theme: ThemeType }) => {
-  const { cancelLabel, theme, ...otherProps } = props
+const MaterialHeaderButtons = (props: {| cancelLabel: string, children: React.Node, theme: ThemeType |}) => {
+  const { cancelLabel, theme } = props
   return (
     <HeaderButtons HeaderButtonComponent={MaterialHeaderButton}
                    OverflowIcon={<MaterialIcon name='more-vert' size={23} color={theme.colors.textColor} />}
                    onOverflowMenuPress={onOverflowMenuPress(cancelLabel)}
-                   {...otherProps}
```

Comment:
are you sure, this was never used?

Reply:
Nope, I readded this in the 629 PR which will be merged at the same time